src/XrdCl/XrdClUtils.hh

```diff
@@ -31,6 +31,7 @@
 #include "XrdCl/XrdClPostMaster.hh"
 #include "XrdCl/XrdClXRootDTransport.hh"
 #include "XrdNet/XrdNetUtils.hh"
+#include "XrdUtils/XrdUtils.hh"
```

Comment:
I see an introduction of a whole new utils scheme here. In general, utility functions should go into one of the established locations. In this case XrdOucUttils.hh and just add a static method to the class to avoid a namespace implementation. That, of course, leads to the next issue is that we would (in either case) have to implementation of this utility method -- one in XrdOucUtils (formerly XrdUtils) and one in XrdClUtils. The consensus is that the best solution is to convert usage of XrdClUtils::plitString to use the one in XrdOucUtils -- messy but fortunately it would only involve name changes. There are other alternatives here like adding a method to XrdOucString that does what splitstring does as there is already a similar method there called tokenize().

Reply:
Ah yes, so this I doubted when I did it... I did not know the existence of the XrdOucUtils.
I wanted the code to not be duplicated. I can remove my class and put it into the XrdOucUtils if you want.
For the XrdOucString, is there a wish that one day, this class is replaced by the std::string ? If yes, I believe we can keep the implementation of the `splitString(...)` method in XrdOucUtils :man_shrugging: